### PR TITLE
Some more 2.1.45 updates

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/editing.md [hunk 1]
```diff
@@ -235,10 +232,16 @@ about media, please see the [media](media.md) section.
 The microphone icon allows you to record from your computer's microphone
 and attach the recording to the note.
 
-The last button shows more advanced features, such as editing the
-underlying HTML of a field, and shortcuts to add MathJax or
+The Fx button shows shortcuts to add MathJax or
 [LaTeX](math.md) to your notes.
 
+The </> button allows editing the underlying HTML of a field.
```
Comment (dae) [inline src/editing.md:238]:
```suggestion
The `</>` button allows editing the underlying HTML of a field.
```

File: src/editing.md [hunk 2]
```diff
@@ -235,10 +232,16 @@ about media, please see the [media](media.md) section.
...
+
+Anki 2.1.45+ supports Sticky Fields natively. If you click on the pin icon on the right of the fields
```
Comment (dae) [inline src/editing.md:240]:
```suggestion
Anki 2.1.45+ supports Sticky Fields natively. If you click on the pin icon on the right of a field
```

File: src/editing.md [hunk 3]
```diff
@@ -235,10 +232,16 @@ about media, please see the [media](media.md) section.
...
-underlying HTML of a field, and shortcuts to add MathJax or
+The Fx button shows shortcuts to add MathJax or
 [LaTeX](math.md) to your notes.
 
+The </> button allows editing the underlying HTML of a field.
+
+Anki 2.1.45+ supports Sticky Fields natively. If you click on the pin icon on the right of the fields 
+, Anki will not clear out this field's content after a note is added. 
+If you find yourself entering the same content into multiple notes, you may find this useful.
+Sticky fields can be toggled on/off from the editing screen.
```
Comment (dae) [inline src/editing.md:243]: This line seems unnecessary?

File: src/preferences.md [hunk 4]
```diff
@@ -48,6 +48,14 @@ note type (and opens the add window to the last-used note type when you choose
 Add). This may be more convenient if you always use a single note type for each
 deck.
 
+**Default search text**
+It allows to customize the starting search text in the browser (eg, to start
```
Comment (dae) [inline src/preferences.md:52]:
```suggestion
Allows you to customize the starting search text in the browser (eg, to start 
```

File: src/studying.md [hunk 5]
```diff
@@ -187,6 +187,8 @@ study, and you can search for flagged cards in the Browse screen. This is useful
 when you want to take some action on the card at a later date, such as looking
 up a word when you get home.
 
+   If you're using Anki 2.1.45+, you can also rename flags from the [browser](browsing.md).
```
Comment (dae) [inline src/studying.md:190]: Unnecessary spaces at the start?

**Review by dae — COMMENTED** [review]
Sorry for the slow reply cqg. A couple of minor tweaks and then this should be good to merge in.

**Comment by cqg2** [comment]
> Sorry for the slow reply cqg. A couple of minor tweaks and then this should be good to merge in.

No problem at all. Thanks, as always for your time!

**Comment by dae** [comment]
Thanks!

**Pull request merged**